lib/heartcheck/checks/base.rb

```diff
@@ -114,11 +114,9 @@ def services
       # @return [Hash]
       def check
         validation
-        hash = { name => { 'status' => (@errors.empty? ? 'ok' : 'error') } }
-        hash[name]['message'] = error_message unless @errors.empty?
-
-        Logger.info Oj.dump(hash)
```

Comment:
Some times is a good ideia log the result to get a history of checks.

With it we can see when an app stop to work and return again.